docs/frontend/webpack.md

```diff
@@ -0,0 +1,55 @@
+# Using Webpack Encore
+
+Webpack Encore is a simpler way to integrate Webpack into your application. 
+It wraps Webpack, giving you a clean & powerful API for bundling JavaScript modules, 
+pre-processing CSS & JS and compiling and minifying assets.
+
+
+### Js
+
+There is an entry for `app.js`. Each entry in `webpack.config.js` will result in one JavaScript file and compiled in `public/build`
+
+You can reference an entry in your templates by name
+
+    {{ encore_entry_script_tags('app') }}
+
+
+### SASS
+
+There is an entry for `style.scss` and `style-dark.scss`. 
+Each entry in `webpack.config.js` will result in one CSS file and compiled in `public/build`
+
+You can reference an entry in your templates by name
+
+    {{ encore_entry_link_tags('style') }}
+    {{ encore_entry_link_tags('style-dark') }}
+
+
+### Images
+
+Image can be placed in `assets/images`. The files wil be moved when compiling to `public/build/images/`.
+A random hash will be added to the file name to prevent browser caching.
+
+
+You can use the `asset`-method in twig templates like below:
+
+    {{ asset('build/images/logo-application.svg') }}
```

Comment:
Eventueel hier vermelding maken van absolute_url()? Want we gebruiken dat wel vaak samen he? asset() geeft relatief pad terug. absolute_url() er rond geeft absoluut pad.

Reply:
Fixed in 2d65d72